src/avatar.jsx

```diff
@@ -24,12 +24,44 @@ const Avatar = React.createClass({
   },
 
   propTypes: {
+    /**
+     * The backgroundColor of the avatar. Does not apply to image avatars.
+     */
     backgroundColor: React.PropTypes.string,
+
+    /**
+     * Can be used, for instance, to render a letter inside the avatar.
+     */
     children: React.PropTypes.node,
+
+    /**
+     * The css class name of the root `div` element.
+     */
+    className: React.PropTypes.string,
```

Comment:
I don't mind this, but I think it helps readability to be more explicit. If you agree please add className to props destructuring and to the div's props.